mythtv/programs/mythfrontend/playbackbox.cpp

```diff
@@ -1981,205 +1995,27 @@ bool PlaybackBox::UpdateUILists(void)
             while (query.next())
             {
                 int recid = query.value(0).toInt();
-                recType[recid] = query.value(1).toInt();
-                maxEpisodes[recid] = query.value(2).toInt();
-                avgDelay[recid] = query.value(3).toInt();
-
-                QDateTime next_record =
-                    MythDate::as_utc(query.value(4).toDateTime());
-                QDateTime last_record =
-                    MythDate::as_utc(query.value(5).toDateTime());
+
                 QDateTime last_delete =
                     MythDate::as_utc(query.value(6).toDateTime());
```

Comment:
Since you aren't using the other values any more I think you could simplify the SQL now to just `SELECT recordid, last_delete FROM record;` and this would become `value(1)`.

Reply:
Oh! Good point. I will sort that out.

Reply:
Okay. I've simplified the SQL, as you suggested.